exercises/concept/lucians-luscious-lasagna/tests/Tests.hs

```diff
@@ -0,0 +1,13 @@
+import Test.Hspec             (Spec, it, shouldBe, hspec)
```

Comment:
this file never uses Spec, so it could (and should) be removed

Reply:
Thanks.